### ISSUE TITLE
Git Workflow License Test and Testing workflow

### ISSUE THREAD

**Issue by thogebati** [amosproj/amos2023ws01-ticket-chat-ai#66]
## Acceptance criteria
- [x] All license are checked correctly
- [x] The tests are executed correctly

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
